lib/yabeda/rails.rb

```diff
@@ -51,7 +51,7 @@ def install!
               status: event.payload[:status],
               format: event.payload[:format],
               method: event.payload[:method].downcase,
-            }.merge!(event.payload.slice(*Yabeda.default_tags.keys))
+            }.merge!(event.payload.slice(*Yabeda.default_tags.keys.excluding(:controller)))
```

Comment:
Note: excluding is only rails 6+.

Reply:
Oh, now it is obvious that we need to exclude all keys that we manually set in. Please try something like this:
```suggestion
            }
            labels.merge!(event.payload.slice(*Yabeda.default_tags.keys - labels.keys))
```